server/api/git/resources.py

```diff
@@ -1,9 +1,26 @@
 from flask.views import MethodView
+from flask import request
+
+from server.services.git.git_service import GitService
+from server.models.serializers.document import DocumentSchema
+from server.services.git.file_service import FileServiceError
 
 
 class GitDocumentApi(MethodView):
     def post(self):
-        return {"msg": "success"}, 201
+        try:
+            document_schema = DocumentSchema()
+            document = document_schema.load(request.json)
+
+            platform_name = document["platform"]["name"]
+            organisation_name = document["organisation"]["name"]
+            project_id = document["project"]["project_id"]
+
+            git_report = GitService(platform_name, organisation_name, project_id)
+            git_report.create_document(document)
+            return {"Success": f"Document for project {project_id} created"}, 201
```

Comment:
Instead of returning "Success" and "Error", I think it's better to return {"detail": "message"}, the status code already give us the information if the request was successful or not.

Reply:
Alright!
I will change to

{"detail": "message"}